www-servers/h2o/files/h2o-fix_help_path.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/main.c
++++ b/src/main.c
+@@ -2015,7 +2015,7 @@
+                        "                               the errors are logged to the file instead of\n"
+                        "                               being emitted to STDERR\n"
+                        "                     - master: invoked process becomes a master process (using\n"
+-                       "                               the `share/h2o/start_server` command) and spawns\n"
++                       "                               the `/usr/share/h2o/start_server` command) and spawns\n"
```

Comment:
You're not going to like it but scripts don't belong in /usr/share. This directory is strictly for data files.